src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,79 @@
 package com.theironyard.novauc;
 
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+    public static Scanner scanner = new Scanner(System.in);
+    public static Scanner scannerInt = new Scanner(System.in);
+
+    public static void main(String[] args) throws Exception {
+
+        ArrayList<InventoryItem> myJunk = new ArrayList<>();
+        InventoryItem kiosk = new InventoryItem();
+
```

Comment:
Octavio nicely done.  Line 14 is not needed because you are never using the InventoryItem that you are creating.  Good job on this assignment.  Keep up the good work.